mobile/src/main/java/rs/readahead/washington/mobile/views/fragment/feedback/SendFeedbackViewModel.kt

```diff
@@ -0,0 +1,102 @@
+package rs.readahead.washington.mobile.views.fragment.feedback
+
+import android.annotation.SuppressLint
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+import dagger.hilt.android.lifecycle.HiltViewModel
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.schedulers.Schedulers
+import rs.readahead.washington.mobile.bus.SingleLiveEvent
+import rs.readahead.washington.mobile.data.database.DataSource
+import rs.readahead.washington.mobile.domain.entity.feedback.FeedbackInstance
+import rs.readahead.washington.mobile.domain.exception.NoConnectivityException
+import rs.readahead.washington.mobile.domain.repository.feedback.FeedBackRepository
+import timber.log.Timber
+import javax.inject.Inject
+
+
+@HiltViewModel
+class SendFeedbackViewModel @Inject constructor(private val feedbackRepository: FeedBackRepository, private val dataSource: DataSource) : ViewModel() {
```

Comment:
I don't understand why you are injecting dataSource here and inside the repository.